templates/kapeta/block-type-cli/package.json.hbs

```diff
@@ -0,0 +1,62 @@
+//#FILENAME:package.json:merge
+{
+    "name": "@{{lowercase data.metadata.name}}",
+    "version": "0.0.1",
+    "description": "",
+    "main": "dist/src/index.js",
+    "type": "commonjs",
+    "scripts": {
+        "prepare:hosts": "node -r ts-node/register ./scripts/prepare-hosts.ts",
+        "prebuild": "cpx \"./package.json\" dist/src",
+        "build": "tsc",
+        "postbuild": "pkg package.json",
+        "predev": "npm run prepare:hosts",
+        "dev": "ts-node -r tsconfig-paths/register src/index.ts",
```

Comment:
We usually call this "start:dev" - and are you not missing "npm run prepare-hosts &&" at the start?

Reply:
`npm run prepare-hosts` runs automatically because I put it in the `"predev"` script.

I though about `"start:dev"` and since it is a CLI that runs once and not a service that is started I thought the word _start_ was a bit misleading. But can add it to be consistent with the other services! 👍